src/ApiService/ApiService/onefuzzlib/NodeOperations.cs

```diff
@@ -113,7 +113,14 @@ public NodeOperations(ILogger<NodeOperations> log, IOnefuzzContext context)
 
             var r = await _context.VmssOperations.UpdateScaleInProtection(nodeInfo.Scaleset, instanceId, protectFromScaleIn: true);
             if (!r.IsOk) {
-                _logTracer.LogOneFuzzError(r.ErrorV);
+                switch (r.ErrorV.Code) {
+                    case ErrorCode.SCALE_IN_PROTECTION_UPDATE_ALREADY_IN_PROGRESS:
```

Comment:
Note: `ErrorCode.SCALE_IN_PROTECTION_INSTANCE_NO_LONGER_EXISTS` is intentionally reported as an error in `AcquireScaleInProtection` and not in `ReleaseScaleInProtection`.

The purpose of releasing scale-in protection is so that azure can delete the node, therefore it's ok if it was already deleted.